src/Persistence/MassTransit.DynamoDbIntegration/DynamoDbIntegration/Saga/DynamoDbDatabaseContext.cs

```diff
@@ -35,7 +35,15 @@ public Task Insert(TSaga instance)
 
         public async Task<TSaga> Load(Guid correlationId)
         {
-            var value = await _database.LoadAsync<DynamoDbSaga>(_options.FormatSagaKey(correlationId), DynamoDbSaga.DefaultEntityType, _options.Config);
+            var value = await _database.LoadAsync<DynamoDbSaga>(_options.FormatSagaKey(correlationId), DynamoDbSaga.DefaultEntityType, new LoadConfig
```

Comment:
In V4 we obsoleted the versions that take the generic OperationConfig because it caused users confusion on what property to set. Now each of the operation has there own specific config.